.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Release information
+
+Please **specify** p/m/M/- for patch/minor/major/none
+Release:
+Please **specify** only ticket id's separated with commas (F.E. BNPL-21, BNPL-22)
+Tickets:
```

Comment:
**question:** How about the cases we want to join more than one PR in a single new release?

Reply:
You can specify - for Release, so it won't trigger release, or leave release field empty, once you will have last PR which should trigger release you can mention release version and all tickets that you are including in scope of new release.